readme.md

```diff
@@ -110,6 +110,8 @@ I'm working towards this however in the meantime the following deployment method
 ```bash
 aws s3 mb s3://mr-virtual-concierge-sam-dev
 
+aws s3 put-bucket-lifecycle-configuration --bucket mr-virtual-concierge-sam-dev --lifecycle-configuration  file://lifecycle.json
```

Comment:
> --bucket mr-virtual-concierge-sam-dev

This was my fault we are now using `virtual-concierge-dev` - I forgot to update the documentation